docs/user/analytics.rst

```diff
@@ -1,5 +1,8 @@
-Traffic Analytics
-=================
+How To Use Traffic Analytics
```

Comment:
If we're not using the `How To` in all the links to this section, why don't we just remove it from the title? It feels odd to only use it here..

Reply:
This is revisiting an old decision. I followed this pattern :100: in all changes, so I don't think it's nice to have to go and undo this pattern now.

There is a note already about supporting a better title/nagivation title/intro in https://github.com/readthedocs/meta/discussions/74

See: https://github.com/readthedocs/readthedocs.org/pull/9620